### ISSUE TITLE
Enchant bug after death

### ISSUE THREAD

**Issue by higuysitme** [FeenixServerProject/Phoenix_1.12.1_Issue_tracker#116]
If you die you lose enchant effect from your legs or head Im not sure which exact one but you lose one of them. Example: I have 8str on my legs and head = 16str total, if I die I get only 8str, same goes with any  zg class enchant.  In order to get it fixed you have to relog every time after death and its kinda frustrating when 40 people supposed to do it after wipe. It is pretty old bug, would be great if you fix this simple bug.

**Comment by Ifixstuffk** [comment]
#122 sorry I didn't see this

**Comment by Yiemeh** [comment]
Nah, the actual bug is you get an "extra" enchant upon login, which is removed once you die. At least this was the case on old feenix 1.12. Can be abused and should be fixed.

Affected enchants are all head/legs enchants and shoulder enchants from sapphiron (ZG shoulder enchs are not affected by this bug)

**Comment by Ashfeenix** [comment]
There are a few issues that can occur here. One such case is that your character LOSES stats until next relog, and another is that you gain an extra enchant upon relogging.

**Replication:**

Create a priest.
.additemset 211 (Transcendence)
.additem 11642 2 (+100 Hp enchants)

Equip full Trans, your hp should be 2857 (give or take 10-20 points depending on the race you chose). Add the enchants to your Helm/Pants, putting you at 3057.

.die targetting yourself. While dead, log out and log back in. .revive yourself. You will either have 100 additional HP (3157) or in some circumstances 200 less than your original value of 2857 (2657). This could be related to a similar issue wherein for a period of time Druids +100 mana enchant was having the same effect.

**Comment by Yiemeh** [comment]
Might be correct I haven't tried it recently. However, the "bonus" upon login was consistent on old Warsong, and the bug appeared sometime in the summer of 2014, if that is of any help.

Edit: at least it was consistent if you were _alive_ when you relogged.
